store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/FederatedGraphStorage.java

```diff
@@ -253,4 +254,34 @@ private boolean isValidToView(final User user, final FederatedAccess access) {
                 .filter(entry -> isValidToView(user, entry.getKey()))
                 .flatMap(entry -> entry.getValue().stream());
     }
+
+    /**
+     * @param user to match visibility against.
+     * @return the set of {@link StoreTrait} that are common for all visible graphs
+     * @see #get(User, List)
+     */
+    public Set<StoreTrait> getTraits(final User user) {
+        return getTraits(user, null);
+    }
+
+    /**
+     * returns a set of {@link StoreTrait} that are common for all visible graphs.
```

Comment:
@p013570 
❓  Is "Min" correct and desired? or "Sum" the goal

Reply:
Yeah I think min is correct.